### PR TITLE
clean up sysctl code

### PR DESCRIPTION
**Pull request by CaoShuFeng**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```
NONE
```

### CODE REVIEW

File: pkg/kubelet/sysctl/runtime.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func NewRuntimeAdmitHandler(runtime container.Runtime) (*runtimeAdmitHandler, er
 			return nil, fmt.Errorf("failed to get runtime version: %v", err)
 		}
 
-		// only Docker >= 1.12 supports sysctls
+		// only Docker API version >= 1.24 supports sysctls
```
Comment (CaoShuFeng) [inline pkg/kubelet/sysctl/runtime.go]: This is consistent with line 31, make the code easy to understand.

File: pkg/kubelet/sysctl/whitelist.go [hunk 2]
```diff
@@ -130,7 +121,7 @@ func (w *patternWhitelist) validateSysctl(sysctl string, hostNet, hostIPC bool)
 	return fmt.Errorf("%q not whitelisted", sysctl)
 }
 
-// Admit checks that all sysctls given in a v1.SysctlsPodAnnotationKey annotation
```
Comment (CaoShuFeng) [inline pkg/kubelet/sysctl/whitelist.go]: Not only v1.SysctlsPodAnnotationKey, but also v1.UnsafeSysctlsPodAnnotationKey.

Comment (feiskyer) [reply pkg/kubelet/sysctl/whitelist.go]: s/in an annotationKey annotation/in annotations v1.SysctlsPodAnnotationKey and v1.UnsafeSysctlsPodAnnotationKey/?

Comment (CaoShuFeng) [reply pkg/kubelet/sysctl/whitelist.go]: Done.

**Review by feiskyer — APPROVED** [review]
LGTM

**Pull request merged**